src/CodeTracking.jl

```diff
@@ -114,7 +114,9 @@ Return the signatures of all methods whose definition spans the specified locati
 Returns `nothing` if there are no methods at that location.
 """
 function signatures_at(filename::AbstractString, line::Integer)
-    filename = abspath(filename)
+    if !startswith(filename, "REPL[")
```

Comment:
Should we do something more robust, e.g. check no file with that name exists first?
And/or check the name follows `r"^REPL@\[\d+\]$"`

Reply:
Good point. I think I favor "or" rather than "and". If there's a file with that name, the user can always force it to choose the file by using an absolute path, whereas you can't force the REPL if you give precedence to files.